### ISSUE TITLE
Unuable to remove the last layers of a model

### ISSUE THREAD

**Issue by uricohen** [szagoruyko/loadcaffe#28]
I am using the caffe reference network as a feature-extractor and would like to remove the last softmax layers (and linear layer `fc8`). It seems that editing the `deploy.prototxt` file to remove it don't work, and it is always added back into `deploy.prototxt.lua`, presumably because it exists in the binary model.

Does this make sense? Is there a way to avoid this? Thanks!

**Comment by uricohen** [comment]
I worked-around this by allowing for manually changing the `deploy.prototxt.lua` file. See pull request:
https://github.com/uricohen/loadcaffe/pull/1

**Comment by szagoruyko** [comment]
Hi, in torch you can interact with a network loaded with loadcaffe as with an ordinary torch network, to remove a module you can just do `net:remove(9)` to remove the 9th module, or `net:insert(nn.Linear(4096,1000),16)` to insert. Check nn documentation.
But I agree that it would be useful to remove modules from prototxt. Will take a look at that.
Will not merge your PR because it breaks the current behavior when you load networks with different backends.
